pwa/src/templates/landing/LandingTemplate.tsx

```diff
@@ -1,6 +1,5 @@
 import * as React from "react";
 import * as styles from "./LandingTemplate.module.css";
-import { Divider, Heading3 } from "@gemeente-denhaag/components-react";
```

Comment:
You have removed this import but forgot to remove the `Divider` implementation on line 81.